source/_integrations/openai_conversation.markdown

```diff
@@ -48,6 +48,75 @@ Top P:
 
 {% endconfiguration_basic %}
 
-### Talking to Super Mario over the phone
+## Talking to Super Mario over the phone
 
 You can use an OpenAI Conversation integration to [talk to Super Mario over a classic landline phone](/voice_control/worlds-most-private-voice-assistant/).
+
+## Services
+
+### Service `openai_conversation.generate_image`
+
+Allows you to ask OpenAI to generate an image based on a prompt. This service
+populates [Response Data](/docs/scripts/service-calls#use-templates-to-handle-response-data)
+with the requested image.
+
+| Service data attribute | Optional | Description                                            | Example          |
+| ---------------------- | -------- | ------------------------------------------------------ | ---------------- |
+| `config_entry`         | no       | Integration entry ID to use.                           |                  |
+| `prompt`               | no       | The text to turn into an image.                        | Picture of a dog |
+| `size`                 | yes      | Size of the returned image in pixels, defaults to 512. | 512              |
```

Comment:
```suggestion
| `size`                 | yes      | Width of the returned image in pixels, defaults to 512. | 512              |
```
I'm assuming it's the width. Or are they square by default?

Reply:
Size, it is square :)